tests/tests/test_os_ent_migration.py

```diff
@@ -21,7 +21,7 @@
 from testutils.api.client import ApiClient
 from testutils.infra.cli import CliTenantadm
 import testutils.api.deviceauth as deviceauth_v1
-import testutils.api.deviceauth_v2 as deviceauth_v2
+import testutils.api.devicedevauth as devicedevauth
```

Comment:
Where is this `devicedevauth` coming from?

Reply:
I see you removed it in a later commit, so probably just a temporary hack for search+replace later...

Reply:
I didn't remove it, I've combined _deviceauth.py_ and _deviceauth_v2.py_ into one _deviceauth.py_ file in testutils/api/

Reply:
My point is that `devicedevauth` does did not exist, nor it existed at any point on the Git history. You probably replaced "auth_v2" with "devauth" and hence ended up with this "devicedevauth" here.